sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -245,4 +245,12 @@ export class ConfigService {
       process.env.KEYCLOAK_AUTH_URL,
     ).href;
   }
+
+  /**
+   * Gets the current API version.
+   * @returns current API version.
+   */
+  get getAPIVersion(): string {
```

Comment:
It is already a getter. So the name could be just apiVersion. `get apiVersion()`